From e1a21d7cb6008216b19ef32c09cfe93d62db5ae4 Mon Sep 17 00:00:00 2001
Message-ID: <e1a21d7cb6008216b19ef32c09cfe93d62db5ae4.1737500778.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1737500778.git.sharlatanus@gmail.com>
References: <cover.1737500778.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 20 Jan 2025 21:30:59 +0000
Subject: [PATCH 14/27] gnu: python-poppy: Update to 1.1.2.

* gnu/packages/astronomy.scm (python-poppy): Update to 1.1.2.
[propagated-inputs]: Add python-numexpr.
[native-inputs]: Remove python-h5py, python-pandas, python-scikit-image, and
python-sphinx; add python-wheel.

Change-Id: I87bd37fc3001d6f341ff7b2751d15533f6a0295d
---
 gnu/packages/astronomy.scm | 32 +++++++++++++++-----------------
 1 file changed, 15 insertions(+), 17 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 4e1b091bb21..4cc69af058e 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -4386,36 +4386,34 @@ (define-public python-poliastro
 (define-public python-poppy
   (package
     (name "python-poppy")
-    (version "1.1.1")
-    (source (origin
-              (method url-fetch)
-              (uri (pypi-uri "poppy" version))
-              (sha256
-               (base32
-                "0s8rb61q8dz66s8d3qg44kb6bb5gi40zl41ik9wyccgb4kyf3brp"))))
+    (version "1.1.2")
+    (source
+     (origin
+       (method url-fetch)
+       (uri (pypi-uri "poppy" version))
+       (sha256
+        (base32 "0mvnd9rlglb1cqhaavd2lyxnvi4xmc133x50rzzlh00xn0gyxgfq"))))
     (build-system pyproject-build-system)
     (arguments
      (list
       #:test-flags
       #~(list "--numprocesses" (number->string (parallel-job-count)))))
+    (native-inputs
+     (list python-docutils
+           python-pytest
+           python-pytest-astropy
+           python-pytest-xdist
+           python-setuptools-scm
+           python-wheel))
     (propagated-inputs
      ;; XXX: With python-synphot (marked as optional) package added to the list
      ;; it tries to download from remote host during tests and fails. Overall
      ;; tests take up to 5-8min to pass.
      (list python-astropy
            python-matplotlib
+           python-numexpr
            python-numpy
            python-scipy))
-    (native-inputs
-     (list python-docutils
-           python-h5py
-           python-pandas
-           python-pytest
-           python-pytest-astropy
-           python-pytest-xdist
-           python-scikit-image
-           python-setuptools-scm
-           python-sphinx))
     (home-page "https://poppy-optics.readthedocs.io/")
     (synopsis "Physical Optics Propagation in Python")
     (description
-- 
2.47.1

